site/developer/model-documentation/generate-model-documentation.qmd

```diff
@@ -1,128 +1,77 @@
 ---
 title: "Generate model documentation"
 date: last-modified
+number-sections: true
```

Comment:
I don't think `number-sections: true` really works if numbering starts at `0.1`. I'd either number manually (my preference) or not bother.

<img width="509" alt="image" src="https://github.com/user-attachments/assets/6bc7269c-d404-4cc5-9da6-1a0e3a8fa70b">